Formula/smlpkg.rb

```diff
@@ -0,0 +1,25 @@
+class Smlpkg < Formula
+  desc "Package manager for Standard ML libraries and programs"
+  homepage "https://github.com/diku-dk/smlpkg"
+  url "https://github.com/diku-dk/smlpkg/archive/v0.1.3.tar.gz"
+  sha256 "cfa7eeff951e04df428694fda38917ee2aaaf0532e2d1dbea7ab4c150f4fe2f0"
+  head "https://github.com/diku-dk/smlpkg.git"
+
+  depends_on "mlkit"
+
+  def install
+    system "make", "-C", "src", "smlpkg"
+    system "make", "install", "INSTALLDIR=#{bin}"
```

Comment:
Could you make a pull request upstream for this to start using `DESTDIR` since that's a standard shared between almost all programs?

Reply:
```suggestion
    system "make", "install", "prefix=#{prefix}"
```